io-engine/src/grpc/v0/mayastor_grpc.rs

```diff
@@ -1763,13 +1763,11 @@ impl mayastor_server::Mayastor for MayastorSvc {
                 for r in nexus.children() {
                     match r.get_uuid() {
                         Some(u) => {
-                            replicas.push(
-                                NexusReplicaSnapshotDescriptor {
-                                    replica_uuid: u.to_string(),
-                                    snapshot_uuid: Some(Uuid::new_v4().to_string()), // unique snapshot UUID.
-                                    skip: false,
-                                }
-                            )
+                            replicas.push(NexusReplicaSnapshotDescriptor {
```

Comment:
oh... we have create snapshot on v0?? Shouldn't have!

Reply:
Oh I see this is old old api...

Reply:
Yes, it is the old API.